sensor/AdblockPlugin.js

```diff
@@ -41,12 +41,14 @@ const rclient = require('../util/redis_manager.js').getRedisClient();
 const bone = require("../lib/Bone.js");
 const sem = require('../sensor/SensorEventManager.js').getInstance();
 const util = require('util');
+const platformLoader = require('../platform/PlatformLoader.js');
+const platform = platformLoader.getPlatform();
 
 const fc = require('../net2/config.js');
 
 const featureName = "adblock";
 const policyKeyName = "adblock";
-const configKey = "ext.adblock.config";
+const configKey = "ext.adblock.conf";
```

Comment:
you can use hash `sys:features:config` for feature configs

Reply:
has changed to sys:features:config and fix a bug about overriding the redis data, for example, when call set API the body is {"ads-adv":"on"}, and the redis data is {"ads":"on"}, the origin logic will cause the redis data is {"ads-adv":"on"} however the correct redis data should be {"ads":"on", "ads-adv":"on"}